__tests__/end-to-end/admin-tests/__snapshots__/article-unpublish.test.js.snap

```diff
@@ -1,15 +1,13 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`can unpublish article is article published 1`] = `
-"This article was published on May 31, 2000.
```

Comment:
We should still be testing this just in the new place where it's at though, right?

Reply:
Or if we don't, then maybe we can just remove these assertions, as if they now just check that the publish button says unpublish, and it always says that, that seems kind of pointless

Reply:
I just ran --u on the snapshots here